ecosystem/indexer-grpc/indexer-grpc-utils/src/counters.rs

```diff
@@ -42,6 +43,7 @@ impl IndexerGrpcStep {
             IndexerGrpcStep::CacheWorkerBatchProcessed => "2",
             // Filestore worker steps
             IndexerGrpcStep::FilestoreUploadTxns => "1",
+            IndexerGrpcStep::FilestoreUpdateMetadata => "2",
```

Comment:
how you're logging it, FilestoreUpdateMetadata happens before FilestoreUploadTxns. so switch these two

Reply:
If we switch these we lose historical data. Let me do 1.1 then.